assessment.md

```diff
@@ -93,9 +93,50 @@ Project Management is Covered by Team 2
 ### Interactions
 
 * Interaction with the customer
+  * :disappointed: Disregards the customer's feelings and
+  refuses to take their needs into account
+  * :confused: Regards some of the customer's feelings
+  but fails to take their ideas and needs into account
+  * :neutral_face: Regards most of the customer's feelings
+  and takes some of their ideas and needs into account
+  * :smiley: Regards all of the customer's feelings and
+  incorporates most of their ideas and needs while also communicating challenges
+  * :sunglasses: Regards all of the customer's feelings, improves ideas,
+  incorporates all of their needs, clearly communicates challenges and
+  provides updates on progress
+
 * Interaction with the team leaders
+  * :disappointed: Impolite, impatient and does not respond to team leaders
+  * :confused: Polite yet impatient and is partially responsive to
+  team leaders but does not communicate clearly
+  * :neutral_face: Polite, patient and is mostly responsive to team leaders
+  with moderate communication
+  * :smiley: Polite, patient and is responsive to team leaders with clear
+  communication
+  * :sunglasses: Polite, patient and is responsive to team leaders with clear
+  communication, providing multiple progress updates
```

Comment:
Is this in-person communication? Communication through GitHub or Slack? All of these?

What do you think, is it worthwhile to further clarify the forum for the interaction with the team leader? Is there a way that this clarification could be "pulled out" of this specific point and made more clear at an earlier stage of the document?

Reply:
I added a clarification of communication methods at the bottom of our Interactions subsection. This states that communication can be done in-person, through GitHub or Slack but should in most cases use a combination of all three. Thank you!